test/Elastic.Apm.Tests/BackendCommTests/PayloadSenderTests.cs

```diff
@@ -348,6 +348,20 @@ internal async void FlushInterval_test(TestArgs args, int numberOfEventsToSend)
 			}
 		}
 
+		[Fact]
+		public void Find_a_better_name()
```

Comment:
Obviously, I forgot to find a better name ;-) - will be addressed in the upcoming commit.

Reply:
I thought it's a challenge for me :D

Reply:
The (updated) test method is now in `ApmServerInfoProviderTests.FillApmServerInfo_warns_on_missing_server_version`